api/envoy/config/core/v3/substitution_format_string.proto

```diff
@@ -65,4 +65,12 @@ message SubstitutionFormatString {
   //   empty string, so that empty values are omitted entirely.
   // * for ``json_format`` the keys with null values are omitted in the output structure.
   bool omit_empty_values = 3;
+
+  // Specify a content_type for text_format. This will be ignored for json_format
+  //
+  // .. code-block::
+  //
+  //   content_type: "text/html; charset=UTF-8"
+  //
+  string content_type = 4 [(validate.rules).string = {min_bytes: 1}];
```

Comment:
Do we need this validation  since it is only valid for text_format?

Reply:
Also specify currently supported content type values are `text/html; charset=UTF-8` and `text/plain` defaulted to `text/plain` ?